node/include/cocaine/detail/service/node/slave/machine.hpp

```diff
@@ -108,12 +109,19 @@ class machine_t : public std::enable_shared_from_this<machine_t> {
 
         metrics::shared_metric<metrics::gauge<std::string>> state;
         metrics::shared_metric<metrics::gauge<std::uint64_t>> uptime;
+        metrics::shared_metric<metrics::gauge<double>> load;
 
         metrics_t(context_t& context, std::shared_ptr<machine_t> parent);
     };
 
     std::unique_ptr<metrics_t> metrics;
 
+    typedef metrics::usts::ewma_t ewma_type;
```

Comment:
I have seen it occasionally as a pattern "here and there" in the code. Was it any changes in naming policy recently?